appveyor_after_build.ps1

```diff
@@ -0,0 +1,300 @@
+. .\appveyor_helpers.ps1
+
+####################### Local run (Debug) configuration ###################################
+
+
+# Set this to true to be able to run outside of appveyor - just configure the section below
+$LOCAL_DEBUG_RUN = $false
+
+
+# Set these input variable that would normally come from AppVeyor yml
+if ($LOCAL_DEBUG_RUN)
+{    
+    $env:SQ_Version = "4.5.5"   
+    $env:APPVEYOR_BUILD_FOLDER = "C:\Users\bgavril\source\repos\sonar-msbuild-runner"
+    $env:configuration = "Debug"
+
+    # This script does not build the C# plugin (unlike appveyor) - please provide the path to it for the debug run
+    $localCSharpPluginPath = $env:APPVEYOR_BUILD_FOLDER + "\DeploymentArtifacts\sonar-csharp-plugin-4.2.Debug.bgavril.jar";
+}
+
+####################### Settings ###################################
+
+# URL prefix from where to download SonarQube (nb: building SQ takes too long)
+$sonarQubeDownloadUrlPrefix = "https://sonarsource.bintray.com/Distribution/sonarqube/sonarqube-";
+# Where the SonarQube zip files are downloaded (easy to cache)
+$sonarQubeDownloadPath = "c:\sonarqube\downloads";
+# Path to the SonarQube server - can be cached but then server would not be clean
+$sonarQubeInstallPath = "c:\sonarqube";
```

Comment:
:+1: to not cache - also uploading the cache and then downloading it again at the beginning of the job takes itself some time as well... which is not necessarly faster than downloading SQ from bintray.